autogen/agentchat/contrib/retriever/__init__.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional
+
+AVILABLE_RETRIEVERS = ["lanchedb", "chromadb"]
```

Comment:
Are you OK with it if we don't move qdrant here in this PR? @sonichi 
I prefer do it in this PR.